editor/scene/connections_dialog.cpp

```diff
@@ -1182,7 +1188,7 @@ void ConnectionsDock::_tree_item_selected() {
 	} else {
 		connect_button->set_text(TTR("Connect..."));
 		connect_button->set_button_icon(get_editor_theme_icon(SNAME("Instance")));
-		connect_button->set_disabled(true);
+		connect_button->set_disabled(is_editing_resource);
```

Comment:
This is wrong, should be reverted.
Although I'm not sure if it's even possible to trigger this condition.